gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
```

Comment:
Commenting here since I can't comment directly on the binary file: Can `gfs.f000.grb2` be moved to test resources or removed completely from the project if not being used for tests?